osu.Game/Graphics/Cursor/MenuCursor.cs

```diff
@@ -143,6 +149,10 @@ private void load(OsuConfigManager config, TextureStore textures, OsuColour colo
                 cursorScale = config.GetBindable<double>(OsuSetting.MenuCursorSize);
                 cursorScale.ValueChanged += newScale => cursorContainer.Scale = new Vector2((float)newScale * base_scale);
                 cursorScale.TriggerChange();
+
+                cursorRotate = config.GetBindable<bool> (OsuSetting.CursorRotation);

```

Comment:
why are you doing this in `Cursor` and not `MenuCursor`?

Reply:
i have no idea how to get OsuConfigManager instance...

Reply:
with dependency injection
(look at other examples)

Reply:
you used it in `DetailSettings` 🤔

Reply:
yeah, noticed and moved, now testing...

Reply:
isn't that a detail?